tests/acceptance/test_accounting_oracle_negative.py

```diff
@@ -174,84 +177,99 @@ def test_setConsensusContract(accounting_oracle: Contract, aragon_agent: Account
             {"from": aragon_agent},
         )
 
+def test_finalize_upgrade(accounting_oracle: Contract, stranger: Account):
```

Comment:
Could you please clarify what specifically needs to be cleaned up?

This test is expected to function correctly after the vote, and it will throw an error if anyone attempts to call `finalizeUpgrade_v2`. Ideally, the test should be removed only when the `finalizeUpgrade_v2` method is removed from the `AccountingOracle` contract in a future upgrade.